extension/src/shared/ui-components/Modal.tsx

```diff
@@ -0,0 +1,60 @@
+import ClearOutlinedIcon from "@mui/icons-material/ClearOutlined";
+import { Box, IconButton, Modal, Typography, styled } from "@mui/material";
+import React, { ReactNode } from "react";
+
+type Props = {
+  title?: string;
+  children?: ReactNode;
+  isVisible: boolean;
+  onClose?: () => void;
+};
+
+const SharedModal: React.FC<Props> = ({ title, isVisible, children, onClose }) => {
+  return (
+    <Modal
+      open={isVisible}
+      aria-labelledby="modal-modal-title"
+      aria-describedby="modal-modal-description">
```

Comment:
We don't have the description of the modal, so you can remove this line.
```suggestion
>
```